src/utils/imgutils.py

```diff
@@ -193,10 +193,13 @@ def draw_template_layout(img, template, shifted=True, draw_qvals=False, border=-
                         2,
                     )
         if shifted:
+            text_in_px = cv2.getTextSize(
+                q_block.key, cv2.FONT_HERSHEY_SIMPLEX,
+                constants.TEXT_SIZE, 4)
             cv2.putText(
                 final_align,
-                "s%s" % (shift),
-                tuple(s - [template.dimensions[0] // 20, -d[1] // 2]),
+                "%s" % (q_block.key),
+                (int(s[0] + d[0] - text_in_px[0][0]), int(s[1] - 20 * constants.TEXT_SIZE)),
```

Comment:
What is this hardcoded 20 about? If it's static, maybe put it in a constant and use it.

Reply:
The 20* text_size is just an offset value.
![image](https://user-images.githubusercontent.com/89850018/193902903-0db5c0d0-8874-44ca-a89c-aaf7718ab020.png)
with offset,
![image](https://user-images.githubusercontent.com/89850018/193903151-51c7c237-28d1-4482-879b-5c7b9799c87c.png)
without offset.
I'll update this since I am now able to give offset using the getTextSize function as shown below
![image](https://user-images.githubusercontent.com/89850018/193903455-04768c6f-6d76-4d84-b193-37a544e80d1e.png)